### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by MukilMuki**
changed all the review comments

### CODE REVIEW

File: Assignment-1/Exercise4.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+public class Exercise4 {
+	static int prim1;
+	static boolean prim2;
+	static double prim3;
+	static float prim4;
+	static long prim5;
+	static String prim6;
+	public static void main(String args[]) {	
+	System.out.println(prim1+" "+prim2+" "+prim3+" "+prim4+" "+prim5+" "+prim6);
```
Comment (karthik-kantam1117) [inline Assignment-1/Exercise4.java]: use proper naming conventions for variables

File: Assignment-1/Exercise6.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	static int armstrong(int num) {
+		int rem,rev=0;
+		while(num>0){
+	          
+	        rem = num%10;
+	        rev += Math.pow(rem,3) ;
+	        num = num/10;
+	      }
+		return rev;
+
```
Comment (karthik-kantam1117) [inline Assignment-1/Exercise6.java]: correct Armstrong number logic --> Math.pow(rem,No.of.digits)

File: Assignment-1/Exercise8.java [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+class Circle implements ShapeArea{
+	
+		public double area(double radius) {
+			double result = 3.14*Math.pow(radius,2);
+		return result;
+		}
+	@Override
+		public double area(double length, double breadth) {
+		return 0;
+		}
```
Comment (karthik-kantam1117) [inline Assignment-1/Exercise8.java]: Add @override and format you code before using to Github

**Review by karthik-kantam1117 — CHANGES_REQUESTED** [review]

**Review by MukilMuki — COMMENTED** [review]
Corrected the changes

**Review by karthik-kantam1117 — DISMISSED** [review]
Satisfied with the review changes.

**Review by karthik-kantam1117 — APPROVED** [review]
Satisfied with the review changes.

**Pull request merged**